src/main/java/sirius/biz/jobs/batch/BatchProcessJobFactory.java

```diff
@@ -170,6 +170,7 @@ protected String getPenaltyToken() {
      */
     protected void executeTask(ProcessContext process) throws Exception {
         try (BatchJob job = createJob(process)) {
+            logParameters(process);
```

Comment:
could be done before creating the job as this might already fail due to invalid parameters